zipline/finance/metrics/metric.py

```diff
@@ -528,3 +529,45 @@ def end_of_bar(self, packet, *args):
         packet['cumulative_risk_metrics']['period_label'] = self._label
 
     end_of_session = end_of_bar
+
+
+class ClassicRiskMetrics(object):
+    """Produces original risk packet
+    """
+
+    def start_of_simulation(self,
+                            ledger,
+                            emission_rate,
+                            trading_calendar,
+                            sessions,
+                            benchmark_source):
+        self._sessions = sessions
+        self._leverages = pd.Series(data=np.NAN, index=sessions)
+        self._daily_benchmark_returns = benchmark_source.daily_returns(
+            sessions[0],
+            sessions[-1],
+        )
+
+    def end_of_session(self,
+                       packet,
+                       ledger,
+                       dt,
+                       next_session_ix,
```

Comment:
I think I am going to change this to `session_ix` just to make this more clear to consumers of this API. I am just going to confirm that the additions don't kill us, but I doubt they will.